cla_backend/apps/legalaid/models.py

```diff
@@ -763,6 +763,24 @@ def is_part_of_split(self):
         """
         return self.from_case or self.split_cases.count() > 0
 
+    def log_denormalized_outcome_fields(self):
+        """
+        For LGA-275 log which fields have been denormalized successfully
+        TODO Replace painfully circuitous reload method when refresh_from_db available in Django 1.8
```

Comment:
📝 Upgrading to Django 1.8 would help with the broken dependencies, too (one of them requires Django 1.8+)